snippets/ponylang-mode/lambda.block

```diff
@@ -3,4 +3,4 @@
 # name: block lambda
 # contributor: Sean T. Allen <sean@monkeysnatchbanana.com>
 # --
-lambda(${1:params})${2:return} => ${3:lambda_block} end$0
\ No newline at end of file
+{(${1:params}) ${2:return} => ${3:lambda_block} } $0
```

Comment:
this is a good change, can you do this in a new PR?

Reply:
yes